Monika After Story/game/script-topics.rpy

```diff
@@ -4940,3 +4940,297 @@ label monika_asks_family:
                 m 1f "You can tell me about your family when you're ready, [player]." 
                 m 1j "I love you very much!"                              
     return
+    
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_fun_facts_open",category=['misc'],prompt="Fun facts",pool=True))
+
+default fun_facts_start = 0
```

Comment:
It looks like `fun_facts_start` just flips between 1 and 0. A boolean would be a better choice.